app/views/dashboards/_table_content.html.erb

```diff
@@ -47,4 +47,7 @@
       <% end %>
     </tbody>
   </table>
+  <% if table_type == 'call_list' && current_user.patients.count > 0%>
+    <%= link_to "Remove all patients from your call list", remove_all_patients_path(current_user), class:"remove-all-patients-link", data: { confirm: "Are you sure you want to remove all your calls?" }, method: :patch, remote: true %>
```

Comment:
Let's change link text to `Clear your call list` and the alert text to `Are you sure you want to clear your current call list?`